manual/hs/manual/manual.cabal

```diff
@@ -52,9 +52,11 @@ executable run-manual
   -- Internal.
   build-depends:
     , c-expr-runtime
+    , ghc-prim
     , hs-bindgen-runtime
     , hs-game
     , hs-vector
+    , primitive
```

Comment:
It would be great if we could actually use the `Prim` instance for a struct in the manual code; one example would suffice; perhaps it would be nice if we would actually show a _use case_; for example, a `PrimArray` of structs. If we do that, it would also be nice if we could (briefly) show that in the actual manual text.